.github/workflows/main.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         python -m vulture vulture/ tests/
 
     - name: Run Tests
+      if: ${{ matrix.python-version != 'pypy3' || matrix.os != 'windows-2019' }}
       run: python -m tox -e py
```

Comment:
Maybe we can simply fix the test instead of skipping it. Can you try to use `tox -e pypy3` for Windows with PyPy and `tox -e py` for all other matrix combinations?

Reply:
That didn't work either..

Reply:
Thanks for testing this. I'll remove the test again and then I'll merge this.